recipes/r-winch/meta.yaml

```
@@ -0,0 +1,94 @@
+{% set version = '0.1.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-winch
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/winch_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/winch/winch_{{ version }}.tar.gz
+  sha256: d3bc6e8b77f4aebcf5dbc611d085ed1d5e807437f12afd77e70f909cc0c763c5
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'   # [win]
+
+requirements:
+  build:
+    - {{ stdlib("c") }}  # [not win]
+    - {{ stdlib("m2w64_c") }}  # [win]
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-lifecycle
+    - r-procmaps >=0.0.2
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-lifecycle
+    - r-procmaps >=0.0.2
+
+test:
+  commands:
+    - $R -e "library('winch')"           # [not win]
+    - "\"%R%\" -e \"library('winch')\""  # [win]
+
+about:
+  home: https://r-prof.github.io/winch/
+  dev_url: https://github.com/r-prof/winch
+  license: GPL-3.0-only
+  summary: Obtain the native stack trace and fuse it with R's stack trace for easier debugging
+    of R packages with native code.
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
```

Explicitly bundle license for vendored software (or remove vendoring):

```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
    - src/vendor/libbacktrace/LICENSE
```